examples/simple.rs

```diff
@@ -53,11 +53,9 @@ fn startup(mut net: ResMut<NetworkResource>, args: Res<Args>) {
 }
 
 fn send_packets(mut net: ResMut<NetworkResource>, time: Res<Time>, args: Res<Args>) {
-    if !args.is_server {
-        if (time.seconds_since_startup() * 60.) as i64 % 60 == 0 {
-            info!("PING");
-            net.broadcast(Packet::from("PING"));
-        }
+    if !args.is_server && (time.seconds_since_startup() * 60.) as i64 % 60 == 0 {
```

Comment:
I don't like this change. These conditions are not related.

First there is a test for the context we are in `if !args.is_server`, then in the context there is another decision of what to do.

Reply:
Ok, if this is really important we can disable the lint that changed that code. Or we can keep the modification and put a comment to tell the reader what he could do.